### DISCUSSION TITLE
Названия методов доставки начинаются с СДЕК. Компания называется СДЭК

### DISCUSSION THREAD

**Discussion by Dustor** [cdek-it/wordpress#45] [Ideas]
Опечатка в префиксе методов доставки

**Comment by vermorag** [comment]
Добрый день. Все опечатки вы можете самостоятельно исправлять через [сервис переводов](https://translate.wordpress.org/projects/wp-plugins/cdekdelivery/).

**Reply by Dustor** [reply]
То есть я должен сам лазить по сервисам переводов, чтобы исправить ошибку разработчиков?

**Reply by Dustor** [reply]
Понимаю, вам кажется, что это ерунда какая-то, чего докопался, работать мешает. Но компания правильно называется не СДЕК, и некультурненько получается )))

**Reply by vermorag** [reply]
В связи с особенностями переводов в Wordpress, любой человек может вносить изменения в перевод. Этот процесс не под контролем разработчиков полностью.
